public/index.html

```diff
@@ -76,6 +76,16 @@
         Clear run
       </button>
     </li>
+    <li>
+      <button id="save-run" aria-label="Save the current run to a JSON file">
```

Comment:
I saw there's an early return for saving `{}` when the run is undefined and the user attempts to save, would it be possible to disable this button until the user places the first point in the run?

Reply:
yes